packages/manager/modules/pci/src/projects/project/instances/instance/edit/edit.html

```diff
@@ -216,7 +216,12 @@ <h1>
                     data-ng-if="$ctrl.instance.flavor.id !== $ctrl.editInstance.flavorId"
                 >
                     <span
-                        data-ng-bind=":: $ctrl.imageEditMessage | translate"
+                        class="d-inline-block"
+                        data-translate="pci_projects_project_instances_instance_edit_change_instance_modele_message_part_1"
+                    ></span>
+                    <span
+                        class="d-inline-block"
+                        data-translate="pci_projects_project_instances_instance_edit_change_instance_modele_message_part_2"
```

Comment:
to avoid using `d-inline-block` class, you can use `p` element it's natively block element.

```suggestion
<p
     data-translate="pci_projects_project_instances_instance_edit_change_instance_modele_message_part_1"
></p>
<p
     data-translate="pci_projects_project_instances_instance_edit_change_instance_modele_message_part_2"
></p>
```

Reply:
yes it's correcting fix for this with <p>. I've apply this feedback